libs/payments/cart/src/gql/tax-address.model.ts

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { Field, ObjectType } from '@nestjs/graphql';
+
+@ObjectType({
+  description:
+    'The Cart associated with a customer Subscription Platform checkout.',
+})
+export class TaxAddress {
```

Comment:
Should TaxAddress be in its own `*.model.ts` file? Should `Subscription`, `TaxAmount`, `Invoice`, etc. also have their own `*.model.ts` files, based on the schema discussed in the Tech Spec?

Reply:
Current structure makes sense.